### PR TITLE
Missing required cmd flags

### PR DESCRIPTION
**Pull request by mrVectorz**
Also fixed missing space before "\"

### CODE REVIEW

**Comment by mrVectorz** [comment]
Without the required flags:
```
[root@openshift-lab-0 ~]# podman run --entrypoint performance-profile-creator -v ./must-gathers/:/must-gather:z quay.io/openshift-kni/performance-addon-operator:4.9-snapshot --must-gather-dir-path /must-gather > performance-profile.yaml
Error: missing required flags: --reserved-cpu-count, --mcp-name, --rt-kernel
Usage:
  performance-profile-creator [flags]

Flags:
      --disable-ht                        Disable Hyperthreading
  -h, --help                              help for performance-profile-creator
      --info string                       Show cluster information; requires --must-gather-dir-path, ignore the other arguments. [Valid values: log, json] (default "log")
      --mcp-name string                   MCP name corresponding to the target machines (required)
      --must-gather-dir-path string       Must gather directory path (default "must-gather")
      --power-consumption-mode string     The power consumption mode.  [Valid values: default, low-latency, ultra-low-latency] (default "default")
      --profile-name string               Name of the performance profile to be created (default "performance")
      --reserved-cpu-count int            Number of reserved CPUs (required)
      --rt-kernel                         Enable Real Time Kernel (required)
      --split-reserved-cpus-across-numa   Split the Reserved CPUs across NUMA nodes
      --topology-manager-policy string    Kubelet Topology Manager Policy of the performance profile to be created. [Valid values: single-numa-node, best-effort, restricted] (default "restricted")
      --user-level-networking             Run with User level Networking(DPDK) enabled
```

File: cmd/performance-profile-creator/README.md [hunk 1]
```diff
@@ -44,13 +44,15 @@ Depending on how must-gather directory was setup run the Performance profile Cre
 
 1. Option 1: Using must-gather output dir (obtained after running must gather manually)
    ```bash
-   podman run --entrypoint performance-profile-creator -v /path/to/must-gather-output:/must-gather:z\
-   quay.io/openshift-kni/performance-addon-operator:4.9-snapshot --must-gather-dir-path /must-gather > performance-profile.yaml
+   podman run --entrypoint performance-profile-creator -v /path/to/must-gather-output:/must-gather:z \
+   quay.io/openshift-kni/performance-addon-operator:4.9-snapshot --must-gather-dir-path /must-gather \
+   --reserved-cpu-count 20 --mcp-name worker-cnf --rt-kernel false > performance-profile.yaml
```
Comment (fromanirh) [inline cmd/performance-profile-creator/README.md:49]: I think we should use placeholders. Like:
```
--reserved-cpu-count $COUNT --mcp-name $NAME --rt-kernel $FLAG
```

File: cmd/performance-profile-creator/README.md [hunk 2]
```diff
@@ -44,13 +44,15 @@ Depending on how must-gather directory was setup run the Performance profile Cre
...
    ```
 1. Option 2: Using an existing must-gather tarball which is decompressed to a directory.
    ```bash
    podman run --entrypoint performance-profile-creator -v /path/to/decompressed-tarball:/must-gather:z \
-   quay.io/openshift-kni/performance-addon-operator:4.9-snapshot --must-gather-dir-path /must-gather > performance-profile.yaml
+   quay.io/openshift-kni/performance-addon-operator:4.9-snapshot --must-gather-dir-path /must-gather \
+   --reserved-cpu-count 20 --mcp-name worker-cnf --rt-kernel false > performance-profile.yaml
```
Comment (fromanirh) [inline cmd/performance-profile-creator/README.md:55]: ditto

File: cmd/performance-profile-creator/README.md (hunk 1 above)
Comment (mrVectorz) [reply cmd/performance-profile-creator/README.md:49]: I thought so too, however in the following example "Running Performance Profile Creator using Wrapper script" exact parameters were used, so I just recycled those ones.
```
./hack/run-perf-profile-creator.sh -t must-gather.tar.gz -- --mcp-name=worker-cnf --reserved-cpu-count=20 \
--rt-kernel=false --split-reserved-cpus-across-numa=true --topology-manager-policy=restricted \
--power-consumption-mode=low-latency > performace-profile.yaml
```

If you want I change both, or just the I'm already fixing, or leave it as it.

File: cmd/performance-profile-creator/README.md (hunk 2 above)
Comment (mrVectorz) [reply cmd/performance-profile-creator/README.md:55]: see previous comment on prior example.

File: cmd/performance-profile-creator/README.md (hunk 1 above)
Comment (fromanirh) [reply cmd/performance-profile-creator/README.md:49]: Let's hear what @swatisehgal prefers. I don't have strong opinions either way, but I don't think that setting values without clearly signaling they are examples is a clear benefit

**Comment by swatisehgal** [comment]
I would recommend that we add a section that lists all the command line args available in the Software Software Components

**Comment by mrVectorz** [comment]
@swatisehgal thanks for the input, I have done as requested.
- added the help output
- clarified that the options are only examples

**Pull request merged**